sherpa/astro/io/__init__.py

```diff
@@ -337,6 +361,13 @@ def read_arf(arg):
 
     """
     data, filename = backend.get_arf_data(arg)
+
+    # It is unlikely that the backend will set this, but allow
+    # it to override the config setting.
+    #
```

Comment:
I wish now I'd been a bit-more explicit in this comment: why am I checking for `emin` being set to decide whether to set `ethresh`? I know I was concerned about not making the code too rigid, but I'd need to go and review the `get_arf_data` and `get_rmf_data` routines in both backends to see exactly what they are doing